department-of-veterans-affairs/caseflow
# Ryan/appeals 29955 && Ryan/appeals 29962

Resolves [Reader Search Bar Needs Placeholder Text Removed When Empty](https://jira.devops.va.gov/browse/APPEALS-29955)

Resolves [Tags Filter Search Bar Needs Placeholder Text](https://jira.devops.va.gov/browse/APPEALS-29962)

# Description
The document and tag filters in reader previously had no placeholder text and did not match the uniform look of caseflow's search bars elsewhere. They now have matching placeholder text.

## Acceptance Criteria

## Testing Plan

1. Log into caseflow as a user that can access reader (BVADWISE works)
2. Navigate to Reader
3. Click the Documents filter tag, notice that there is placeholder text in the search bar now.
4. Click the Tags filter tag, notice that there is placeholder text in the search bar now.

# Frontend
## User Facing Changes

 BEFORE|AFTER
 
<img width="236" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/4c008872-5ed0-4f21-b215-9a316e679115">
<img width="247" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/086580ba-cf69-4004-89ac-505abf2a25c8">

|
<img width="265" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/68f5d835-99d8-43a1-a9af-83476d460c61">
<img width="233" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/0a1266ff-1f08-4315-ab6c-a3f25d801af2">
